src/less/less/ds6/mixins.less

```diff
@@ -440,3 +440,71 @@
 .icon-red-laser(@width: @ds6-icon-red-laser-width; @height: @ds6-icon-red-laser-height) {
     .background-icon-image(@width; @height; 'PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHhtbG5zOnhsaW5rPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5L3hsaW5rIiB3aWR0aD0iMjAiIGhlaWdodD0iMTgiIHZpZXdCb3g9IjAgMCAyMCAxOCI+PHBhdGggZmlsbC1ydWxlPSJldmVub2RkIiBkPSJNMy44NDIgNy41OTVIMS43OVYuN2EuNy43IDAgMCAxIC43LS43aC42NTNhLjcuNyAwIDAgMSAuNy43djYuODk1em0wIDIuMDhWMTcuM2EuNy43IDAgMCAxLS43LjdIMi40OWEuNy43IDAgMCAxLS43LS43VjkuNjc2aDIuMDUzem0yLjA1My0yLjA4SDQuODY4Vi41MTNhLjUxMy41MTMgMCAwIDEgMS4wMjcgMHY3LjA4MnptMCAyLjA4djcuODEyYS41MTMuNTEzIDAgMSAxLTEuMDI3IDBWOS42NzZoMS4wMjd6bTEyLjMxNi0yLjA4aC0yLjA1M1YuN2EuNy43IDAgMCAxIC43LS43aC42NTNhLjcuNyAwIDAgMSAuNy43djYuODk1em0wIDIuMDhWMTcuM2EuNy43IDAgMCAxLS43LjdoLS42NTNhLjcuNyAwIDAgMS0uNy0uN1Y5LjY3NmgyLjA1M3ptLTMuMDgtMi4wOGgtMS4wMjZWLjUxM2EuNTEzLjUxMyAwIDEgMSAxLjAyNyAwdjcuMDgyem0wIDIuMDh2Ny44MTJhLjUxMy41MTMgMCAxIDEtMS4wMjYgMFY5LjY3NmgxLjAyN3ptLTYuMTU3LTIuMDhINi45MlYuN2EuNy43IDAgMCAxIC43LS43aC42NTNhLjcuNyAwIDAgMSAuNy43djYuODk1em0wIDIuMDhWMTcuM2EuNy43IDAgMCAxLS43LjdINy42MmEuNy43IDAgMCAxLS43LS43VjkuNjc2aDIuMDUzem0yLjA1Mi0yLjA4SDEwVi41MTNhLjUxMy41MTMgMCAxIDEgMS4wMjYgMHY3LjA4MnptMCAyLjA4djcuODEyYS41MTMuNTEzIDAgMSAxLTEuMDI2IDBWOS42NzZoMS4wMjZ6bTIuMDUzLTIuMDhoLTEuMDI2Vi41MTNhLjUxMy41MTMgMCAxIDEgMS4wMjYgMHY3LjA4MnptMCAyLjA4djcuODEyYS41MTMuNTEzIDAgMSAxLTEuMDI2IDBWOS42NzZoMS4wMjZ6TS43NyA4LjExNmgxOC40NmEuNTIuNTIgMCAxIDEgMCAxLjA0SC43N2EuNTIuNTIgMCAwIDEgMC0xLjA0eiIvPjwvc3ZnPg==');
 }
+
+// Misc
+.placeholder() {
```

Comment:
I'm not sure there's any need to expose these globally. Once we make them available, they become very difficult to remove, so I say keep them internal to textbox module for now. Unless, the new floated label module will be using them?

Also, if a mixin is only ever used once, then just remove it as a mixin (I'm not sure if you plan on reusing them more when it comes to the floating labels).

Reply:
`.placeholder` and `.placeholder-disabled` is used just once but `floated` one bit more. I like to to keep it as mixin, just to have a more cleaner/simpler code.

Reply:
By internal I mean a `textbox-mixins.less`

Reply:
updated.